install/streams.csv

```diff
@@ -915,6 +915,8 @@ HIZ,21,H,N,200,false,false,false,2021-05-23T22:10:00Z,9999-01-01T00:00:00Z
 HLRZ,10,E,H,100,false,false,false,2008-12-03T01:00:00Z,9999-01-01T00:00:00Z
 HMCS,20,B,N,50,true,false,false,2013-06-12T02:00:00Z,2020-06-16T00:00:00Z
 HMCS,20,H,N,200,true,false,true,2001-11-12T00:00:00Z,9999-01-01T00:00:00Z
+HNCZ,10,L,H,1,false,false,false,2023-08-23T23:19:00Z,9999-01-01T00:00:00Z
+HNCZ,10,E,H,100,false,false,false,2023-08-23T23:19:00Z,9999-01-01T00:00:00Z
```

Comment:
@Thomas-Benson think this might have to be 
```
HNCZ,10,**E**,H,100,false,false,false,2023-08-23T23:19:00Z,9999-01-01T00:00:00Z
```
as it is a high gain seismomter now.